src/register.ts

```diff
@@ -31,5 +25,5 @@ export const register = (locale: string, func: LocaleFunc) => {
  * @returns {*}
  */
 export const getLocale = (locale: string): LocaleFunc => {
-  return Locales[locale] || en_US;
+  return Locales[locale] || Locales['en_US'];
```

Comment:
instead of using the imported function I assume that it can be resolved by name and that it's been registered if built-in locales are used.